script/crossbinary-default

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+
+if ! test -e autogen/gen.go; then
+	echo >&2 'error: generate must be run before crossbinary'
+	false
+fi
+
+if [ -z "$VERSION" ]; then
+    VERSION=$(git rev-parse HEAD)
+fi
+
+if [ -z "$CODENAME" ]; then
+    CODENAME=cheddar
+fi
+
+if [ -z "$DATE" ]; then
+    DATE=$(date -u '+%Y-%m-%d_%I:%M:%S%p')
+fi
+
+# Get rid of existing binaries
+rm -f dist/traefik_*
+
+# Build 386 amd64 binaries
+OS_PLATFORM_ARG=(linux windows)
```

Comment:
I'm in favor of including Darwin here: there are a number of Mac developers, and we should be careful not to break the build for them.

In return...